src/ColorBlock.js

```diff
@@ -1,13 +1,15 @@
 import './ColorBlock.css'
 
-export default function ColorBlock(props) {
-  let style = {
-    backgroundColor: props.color
+
+
+export default function ColorBlock({color, boxColor, checkGuess, showHex}) {
```

Comment:
I forgot about destructing props, thanks for the reminder!

Reply:
It's good to have someone look at our code, I always see the opportunity to destruct in other people's code, but often forget about it on my code too